scarb/tests/build_cairo_plugin.rs

```diff
@@ -310,3 +310,98 @@ fn can_emit_plugin_diagnostics() {
             error: could not compile `hello` due to previous error
         "#});
 }
+
+#[test]
+fn can_remove_original_node() {
+    let temp = TempDir::new().unwrap();
+    let t = temp.child("some");
+    simple_project_with_code(
+        &t,
+        indoc! {r#"
+        use cairo_lang_macro::{ProcMacroResult, TokenStream, attribute_macro};
+
+        #[attribute_macro]
+        pub fn some_macro(token_stream: TokenStream) -> ProcMacroResult {
+            let _code = token_stream.to_string();
```

Comment:
```suggestion
        pub fn some_macro(_: TokenStream) -> ProcMacroResult {
```

for fun, to check that we're not relying on this parameter name?